packages/apps/explorer/src/components/AccountBalanceDistribution/components/ChainBalance.tsx

```diff
@@ -0,0 +1,60 @@
+import type { IViewChain } from '@/utils/processChainAccounts';
+import { Stack, Text } from '@kadena/kode-ui';
+import { assignInlineVars } from '@vanilla-extract/dynamic';
+import classNames from 'classnames';
+import type { FC, PropsWithChildren } from 'react';
+import React from 'react';
+import {
+  chainBalanceWrapperClass,
+  chainTextBaseClass,
+  chainTextDisabledClass,
+  chainTextLargeClass,
+  chainTextSubtleClass,
+  percentageValueVar,
+} from './style.css';
+
+interface IProps extends PropsWithChildren {
+  chain: IViewChain;
```

Comment:
chainAccount?

Reply:
?? what do you mean?

Reply:
Instead of `chain` rename the prop name to `chainAccount`